@xen-orchestra/lite/scripts/release.mjs

```diff
@@ -342,11 +352,14 @@ if (ghRelease) {
       stop()
     }
   } else {
+    const releaseNotes = getChangelogForVersion(version)
```

Comment:
I think we should at least prompt something if `releaseNotes` is `null` because it's not expected and it might mean something is wrong with the version numbers.